R/geoms.R

```diff
@@ -102,39 +102,52 @@ uses_geoms <- function(p, geoms, stats = NULL, exact = TRUE) {
   }
 }
 
-#' Does a layer use a specific geom parameter?
+#' Does a layer use a specific parameter?
```

Comment:
Since you can check multiple parameters, maybe this should be:

```suggestion
#' Does a layer use one or more specific parameters?
```

Reply:
Actually, with that in mind, maybe the function should be `uses_geom_params()`?

Reply:
Great points, I've updated to make this change.